wiki/FAQ/en.md

```diff
@@ -281,11 +276,11 @@ _Detailed explanation of scoring can be found at the game style's own respective
   -   If the beatmap has 3 difficulties, and you SS all three of them, you will receive 3 SS and not just one SS in your profile.
 
 ## Performance Points
-*Full explanation: [Performance Points][Performance_Points wikilink]*
+_Full explanation: [Performance Points][Performance Points wikilink]_
 
-Performance Points (pp) is a ranking metric aiming to be more contextually relevant to a continuous game like osu!. It aims to shift the focus from being on the amount of time played to the actual skill of the player. Performance points are heavily based on map difficulty which is determined by a unique algorithm in every gamemode. Based on this difficulty alone each of your scores is rated and assigned a pp value. Your performance rank is predominantly based on your performance on individual maps. The easiest way to improve it is to improve your scores on difficult songs. Do note that game modes are counted separately \[which means, playing (osu!/osu!taiko/osu!catch/osu!mania) raises (osu!/osu!taiko/osu!catch/osu!mania) pp\].
+Performance Points (pp) is a ranking metric aiming to be more contextually relevant to a continuous game like osu!. It aims to shift the focus from being on the amount of time played to the actual skill of the player. Performance points is heavily based on map difficulty, determined by a unique algorithm in every game mode. Based on this difficulty alone, each of your scores will be rated and assigned a pp value. Your performance rank is predominantly based on your performance on individual maps. The easiest way to improve it is to improve your scores on difficult songs. Do note that game modes are counted separately \[which means, playing (osu!/osu!taiko/osu!catch/osu!mania) raises (osu!/osu!taiko/osu!catch/osu!mania) pp\].
```

Comment:
Lower case the P in Performance Points (first one). And change the last sentence to this:
>Do note that game modes are counted separately. This means that playing osu!, osu!taiko, osu!catch, or osu!mania will raise the pp for that game mode.